src/payment/PaymentPage.jsx

```diff
@@ -30,10 +30,7 @@ class PaymentPage extends React.Component {
   }
 
   renderEmptyMessage() {
-    const {
-      dashboardURL,
-      supportURL,
-    } = this.props;
+    const { dashboardURL, supportURL } = this.props;
```

Comment:
Formatted the file which fixed a bunch of indentation and stuff that I hadn't even noticed.  Figured I'd just keep it since I was going back and trying to make things clearer anyway. :)

Reply:
Are these things that are worth putting into our eslint config?

Reply:
Hm, good point - I think my formatter (Prettier) may be taking some extra liberties beyond what our eslint config says.